samcli/commands/delete/delete_context.py

```diff
@@ -4,8 +4,10 @@
 import logging
 
 import json
-import boto3
 
+from typing import Optional
+
+import boto3
```

Comment:
Is this an unused import?

Reply:
The `Optional` class is currently used for the S3 option type as the option could be `None` (ie not specified by the user)

```python
def __init__(
        self,
        stack_name: str,
        region: str,
        profile: str,
        config_file: str,
        config_env: str,
        no_prompts: bool,
        s3_bucket: Optional[str],
        s3_prefix: Optional[str],
):
```

Reply:
even so, the line `import boto3` can be removed, right?

Reply:
@lucashuy Sorry I was referring to import boto3 not Optional!

Reply:
Oh I see, no worries! boto3 is used in that class to create S3 and ECR clients used to delete data, so its another needed import.

Reply:
Oh I see, this is not a new import you just moved the position, got it!